yarn.lock

```diff
@@ -740,6 +740,173 @@ __metadata:
+  languageName: node
+  linkType: hard
+
+"@inquirer/number@npm:^1.0.10":
+  version: 1.0.10
+  resolution: "@inquirer/number@npm:1.0.10"
+  dependencies:
+    "@inquirer/core": ^9.0.10
+    "@inquirer/type": ^1.5.2
+  checksum: c5e900e793ca885c1075f2f1ad2e3d96caba63451ea72bc89d8665dc3bf6a308e28aafa50a61f98c69e6852e9a69d48d5052c325237b774d901f08ae1230f462
+  languageName: node
+  linkType: hard
+
+"@inquirer/password@npm:^2.1.22":
+  version: 2.1.22
+  resolution: "@inquirer/password@npm:2.1.22"
+  dependencies:
+    "@inquirer/core": ^9.0.10
+    "@inquirer/type": ^1.5.2
+    ansi-escapes: ^4.3.2
+  checksum: 0bf6dea2594e2ff87151aa6c99299cc7185e7b0a7d4265d5c28873b37db39ed97820e3d3c8117c8ab1639ce32825ee99a9b2a655bcaa65bbfc0cdd7bd71e8724
+  languageName: node
+  linkType: hard
+
+"@inquirer/prompts@npm:^5.3.2":
+  version: 5.3.8
+  resolution: "@inquirer/prompts@npm:5.3.8"
+  dependencies:
+    "@inquirer/checkbox": ^2.4.7
+    "@inquirer/confirm": ^3.1.22
+    "@inquirer/editor": ^2.1.22
+    "@inquirer/expand": ^2.1.22
+    "@inquirer/input": ^2.2.9
+    "@inquirer/number": ^1.0.10
+    "@inquirer/password": ^2.1.22
+    "@inquirer/rawlist": ^2.2.4
+    "@inquirer/search": ^1.0.7
+    "@inquirer/select": ^2.4.7
+  checksum: 10f82ff86011ebfaa7aa5a8f34dcdbb53f84a10c009e9e41e861c5636bd548cf38c187d5ccd6ea3bbe5f260cdd5023c75e40446973c1a87d62276af230a006a8
+  languageName: node
+  linkType: hard
+
+"@inquirer/rawlist@npm:^2.2.4":
+  version: 2.2.4
+  resolution: "@inquirer/rawlist@npm:2.2.4"
+  dependencies:
+    "@inquirer/core": ^9.0.10
+    "@inquirer/type": ^1.5.2
+    yoctocolors-cjs: ^2.1.2
+  checksum: dd9d34a5cca081d53a9798cdfed2fdb61455dcfa856f54bc036dc5f57aceb95a7484487632c157bdba75e50de24990ebb3bb178ee765b8c0a735ff61b29cebf4
+  languageName: node
+  linkType: hard
+
+"@inquirer/search@npm:^1.0.7":
+  version: 1.0.7
+  resolution: "@inquirer/search@npm:1.0.7"
+  dependencies:
+    "@inquirer/core": ^9.0.10
+    "@inquirer/figures": ^1.0.5
+    "@inquirer/type": ^1.5.2
+    yoctocolors-cjs: ^2.1.2
+  checksum: bfa4f033ca53ed92d904e53903be9e995b56c7308790b95f9f57855b4b6e761a78eb58bcd439ad5dad5bed2e59803d9a6c67b6681bb3f70d297325a0fd62ec4f
+  languageName: node
+  linkType: hard
+
+"@inquirer/select@npm:^2.4.7":
+  version: 2.4.7
+  resolution: "@inquirer/select@npm:2.4.7"
+  dependencies:
+    "@inquirer/core": ^9.0.10
+    "@inquirer/figures": ^1.0.5
+    "@inquirer/type": ^1.5.2
+    ansi-escapes: ^4.3.2
+    yoctocolors-cjs: ^2.1.2
+  checksum: 77d0d708983d4b432cd6bb9e4bc3c53308f3a3c97fa8c688fb2a05a14d3a7211d7cc3f3b72f17f0e1a2879658724184ac25b2da0e2cf5629151b86441567ec64
+  languageName: node
+  linkType: hard
+
+"@inquirer/type@npm:^1.5.2":
+  version: 1.5.2
+  resolution: "@inquirer/type@npm:1.5.2"
+  dependencies:
+    mute-stream: ^1.0.0
+  checksum: 6f5ee8143c982aaab0d14871f346b5770c9630e0f14303119c03fb9cb7904f6e0af0b76dcef9528da16563877f65e80a6056aa6d90e77511ab42177cdf06c4d7
+  languageName: node
+  linkType: hard
```

Comment:
Just curious why we need `inquirer` it seems to useful only when building cli applications